code/__DEFINES/__game.dm

```diff
@@ -90,12 +90,14 @@
 
 //Preference toggles//
 //toggles_sound
-#define SOUND_ADMINHELP	1
-#define SOUND_MIDI		2
-#define SOUND_AMBIENCE	4
-#define SOUND_LOBBY		8
-#define SOUND_INTERNET	16
-#define SOUND_REBOOT	32
+#define SOUND_ADMINHELP 1
+#define SOUND_MIDI 2
+#define SOUND_AMBIENCE 4
+#define SOUND_LOBBY	8
+#define SOUND_INTERNET 16
+#define SOUND_REBOOT 32
+#define SOUND_ADMIN_MEME 64
+#define SOUND_ADMIN_ATMOSPHERIC	128
```

Comment:
why mess up the numbers... also shouldn't this use flags ie 1<<0 1<<1 1<<2 etc

Reply:
i recall walter saying using tabs is bad and we should just do a single space to separate instead

but yes ok